pkg/zabbix_api.go

```diff
@@ -83,9 +84,9 @@ func (ds *ZabbixDatasource) ZabbixAPIQuery(ctx context.Context, tsdbReq *datasou
 		apiMethod := jsonQuery.Get("method").MustString()
 		apiParams := jsonQuery.Get("params").MustMap()
 
-		response, err := ds.ZabbixRequest(ctx, dsInfo, apiMethod, apiParams)
-		ds.queryCache.Set(HashString(tsdbReq.String()), response)
-		result = response
+		var err error
+		result, err = ds.ZabbixRequest(ctx, dsInfo, apiMethod, apiParams)
```

Comment:
I believe as part of this comment https://github.com/alexanderzobnin/grafana-zabbix/pull/815#discussion_r357008668 we removed ":=" which breaks the plugin. But the next commit added this back so went ahead fixed it.